package.xml

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>mqtt_bridge</name>
-  <version>1.2.2</version>
+  <version>1.2.4</version>
```

Comment:
please don't include version changes in a PR, they are done a part of the release process